spec/spec_helper.rb

```diff
@@ -13,7 +13,68 @@
 # it.
 #
 # See http://rubydoc.info/gems/rspec-core/RSpec/Core/Configuration
+require 'webmock/rspec'
+WebMock.disable_net_connect!(allow_localhost: true)
+
 RSpec.configure do |config|
+  config.before(:each) do
+    stub_request(:get, 'https://graph.facebook.com/me?access_token=12345' \
+      "&appsecret_proof=#{OpenSSL::HMAC.hexdigest('SHA256', ENV['APP_SECRET_KEY_FB'], '12345')}" \
+      '&fields=name,email,gender')
+      .with(
+        headers: {
+          'Accept' => '*/*',
+          'Accept-Encoding' => 'gzip;q=1.0,deflate;q=0.6,identity;q=0.3',
+          'User-Agent' => 'Faraday v0.15.3'
+        }
+      ).to_return(
+        status: 200,
+        body: {
+          'id': '111345443186894',
+          'name': 'John Doe',
+          'gender': 'male',
+          'email': 'johnexample@example.com'
+        }.to_json
+      )
+  end
+
+  config.before(:each) do
```

Comment:
You can remove this line and include this stub request in the first `config.before(:each)` block